health.go

```diff
@@ -3,6 +3,7 @@ package health
 import (
 	"context"
 	"fmt"
+	"log"
```

Comment:
So what's this for?

Reply:
I have one place where the library code may create an error that I can't push to the caller, as it's simply a programmer error. The error is logged instead.